frontend/src/features/research/add/AddResearchForm.tsx

```diff
@@ -0,0 +1,98 @@
+import { InlineInput } from 'components/common/form/styles';
+import { Formik } from 'formik';
+import { Api_ResearchFile } from 'models/api/ResearchFile';
+import * as React from 'react';
+import { Button, Col, Row } from 'react-bootstrap';
+import { Prompt, useHistory } from 'react-router-dom';
+import styled from 'styled-components';
+
+import { useAddResearch } from '../hooks/useAddResearch';
+import { AddResearchFileYupSchema } from './AddResearchFileYupSchema';
+import { ResearchForm } from './models';
+import ResearchProperties from './ResearchProperties';
+
+const AddResearchForm: React.FunctionComponent = () => {
```

Comment:
getting a 404 from here: /research/ogs-internal/ows

Reply:
also: 
![image](https://user-images.githubusercontent.com/41091511/162853341-fefd0c2b-2a6a-4387-b376-3f60b3b8584d.png)

Reply:
Found the issue, it turns out the baseAxios was taking the current url as a base if nothing was passed as baseUrl parameter

Reply:
The issue with the icon is still a problem @FuriousLlama